lib/src/ui/columns/pluto_column_title.dart

```diff
@@ -123,25 +123,22 @@ class PlutoColumnTitleState extends PlutoStateWithChange<PlutoColumnTitle> {
     _isPointMoving = false;
   }
 
-  @override
-  Widget build(BuildContext context) {
+  Widget _titleIcons() {
     final style = stateManager.configuration.style;
 
-    final columnWidget = _SortableWidget(
-      stateManager: stateManager,
-      column: widget.column,
-      child: _ColumnWidget(
-        stateManager: stateManager,
-        column: widget.column,
-        height: widget.height,
-      ),
-    );
+    Widget? auxiliarIndicator;
```

Comment:
I just didn't like the `auxiliar` name. :sweat_smile: 

How about `leftIcon` or `leadingIcon`? I don't know either.